sat/cli/bootprep/input/session_template.py

```diff
@@ -121,11 +121,16 @@ def validate_configuration_exists(self, **_):
             return
 
         try:
-            self.cfs_client.get_configuration(self.configuration)
+            resp = self.cfs_client.get('configurations', self.configuration, raise_not_ok=False)
+            if not resp.ok:
+                if resp.status_code == 404:
+                    raise InputItemValidateError(f'Configuration {self.configuration} specified for '
+                                                 f'{self} does not exist.')
+                else:
+                    self.cfs_client.raise_from_response(resp)
```

Comment:
I don't think anything will catch this, will it? That will result in a traceback on an `APIError` rather than a nice validation error to the user. I would suggest that we still raise an `InputItemValidateError` in this case, but just with an error message that says that the query to CFS failed for some reason.

Reply:
This will be caught on line 131 and be re-raised as an `InputItemValidateError` below. This should behave the same way as if a non-404 error was raised previously.